finish/inventory/src/main/java/io/openliberty/guides/inventory/InventoryManager.java

```diff
@@ -28,13 +28,14 @@
 import io.openliberty.guides.inventory.model.*;
 
 @ApplicationScoped
+// tag::InventoryManager[]
 public class InventoryManager {
     
     private List<SystemData> systems = Collections.synchronizedList(new ArrayList<>());
     private SystemClient systemClient = new SystemClient();
-    // tag::custom-tracer[]
+    // tag::customTracer[]
```

Comment:
add blank line above this line so methods are spaced out properly

Reply:
Addressed.